src/hooks.server.ts

```diff
@@ -0,0 +1,13 @@
+
+/** @type {import('@sveltejs/kit').Handle} */
+export async function handle({ event, resolve }: any) {
```

Comment:
If I understand correctly, this relies on dynamic server-side rendering to work, right? Right now callthis is entirely a static site, with no dynamic servers involved. Can we do this client-side instead?

Something like this: https://g.co/gemini/share/11c278cdc33b

Alternatively, we could also use [localStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage).